content/applications/inventory_and_mrp/plm/manage_changes/eco_type.rst

```diff
@@ -39,105 +49,84 @@ the following information:
+Verification stages
+-------------------
 
-To configure a verification stage, hover over the intended stage, and select the :guilabel:`⚙️
-(gear)` icon. Then, click :guilabel:`Edit` to open a pop-up window.
-
-Configure the verification stage in the edit stage pop-up window, by checking the box for
+To configure a verification stage, hover over the intended stage, and select the :icon:`fa-gear`
+:guilabel:`(Actions)`. Then, click :guilabel:`Edit` to open a pop-up window and check the box for
 :guilabel:`Allow to apply changes`.
 
-Then, add an approver in the :guilabel:`Approvers` section, by clicking :guilabel:`Add a line`, and
-specifying the :guilabel:`Role` of the reviewer, their :guilabel:`User`, and :guilabel:`Approval
-Type`.
-
-Make sure at least one approver is configured with the :guilabel:`Approval Type`: :guilabel:`Is
-required to approve`.
+Next, add an approver in the :guilabel:`Approvers` section, by clicking :guilabel:`Add a line`, and
+specifying the :guilabel:`Role` of the reviewer, the :guilabel:`User`, and an :guilabel:`Approval
+Type` of :guilabel:`Is required to approve`. See :ref:`more about approval types.
+<plm/approvals/approval-type>`
 
 The approver listed is automatically notified when |ECOs| are dropped in the stage specified in the
 pop-up window. Once finished, click :guilabel:`Save & Close`.
 
-.. example::
-   In the |ECO| type `New Product Introduction`, the verification stage `Validated` is configured by
-   clicking the :guilabel:`⚙️ (gear)` icon, and selecting :guilabel:`Edit`. Doing so opens the
-   :guilabel:`Edit: Validated` pop-up window.
-
-   By adding the `Engineering manager` as an approver, only |ECOs| approved by this user can proceed
-   to the next stage, and have the changes applied on the production |BOM|.
+.. seealso::
+   :doc:`../management/approvals`
 
-   Additionally, check the :guilabel:`Allow to apply changes` option to ensure proper behavior.
+.. _plm/eco/closing-stage:
 
-   .. image:: eco_type/verification-stage.png
-      :align: center
-      :alt: Show "Allow to apply changes" option is checked.
+Closing stages
+--------------
 
-Closing stage
--------------
+Click an |ECO| type from :menuselection:`PLM app --> Overview` to open the Kanban view of |ECOs| of
+this type.
 
-Configure a closing stage by opening the :guilabel:`Edit: [stage]` pop-up window. To do so, hover
-over the intended stage and click the :guilabel:`⚙️ (gear)` icon that appears in the top-right
-corner. Then, click :guilabel:`Edit` from the drop-down menu.
-
-On the :guilabel:`Edit: [stage]` pop-up window, select the check boxes for :guilabel:`Folded in
-kanban view`, :guilabel:`Allow to apply changes` and :guilabel:`Final Stage`.
+To configure a closing stage that applies |BOM|, edit the stage and tick the boxes for
+:guilabel:`Folded in kanban view`, :guilabel:`Allow to apply changes`,and :guilabel:`Final Stage`.
+|BOM| and operations changes in the |ECOs| will immediately be put into effect. To create a
```

Comment:
Suggestion to add slightly more cause/effect context, beyond configuring the closing stage.
```suggestion
When ECO cards are placed in a stage that will *Allow to apply changes*, then any |BOM| and
operations changes validated in the |ECOs| will immediately be put into effect. To create a
```